### ISSUE TITLE
Authenticator length

### ISSUE THREAD

**Issue by BasileiosKal** [wbl/draft-ladd-bbs-privacypass#1]
A Token has an authenticator of constant length (Nk). For BBS, the authenticator is a proof generated by [ProofGen](https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-bbs-signatures-03#name-proof-generation-proofgen). The proof's size depends on the number of disclosed attributes.

How can this be defined?? 
- Define Nk to be N/A in the IANA registry and the authenticator to be something like `uint8_t authenticator<0..2^32>`?? 
- Remove the authenticator and add a proof field?? 
- Define a constant number of selectively disclosable items, like `[sid, iat, nonce]`?? Or not use attributes at all??

**Comment by wbl** [comment]
Where does this get defined? It's not the in the architecture draft which is the main thing I looked at when writing our text.

**Comment by BasileiosKal** [comment]
Yes sorry! I was going of the [HTTP Authentication](https://www.ietf.org/archive/id/draft-ietf-privacypass-auth-scheme-11.html#name-token-redemption) and [Issuance Protocol](https://www.ietf.org/archive/id/draft-ietf-privacypass-protocol-11.html#name-issuance-protocol-for-publi) documents (was trying out the http protocol).

**Comment by wbl** [comment]
https://www.ietf.org/archive/id/draft-ietf-privacypass-auth-scheme-11.html#section-2.1-5 says tokens may change the type of the message, so I think Nk being NA and changing the type is fine
